_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+Author: Levi Blinder
+Title: Arts and Sciences Recipe Page
+---
+
+```
+<doc>
+...
+...
+...
+<div type = "recipe">
+  ...
+  ...
+  ...
+  <page>44</page>
+  <pageNum>14</pageNum>
+  <p>
+    it down with a <tl> String </tl> as far as
```

Comment:
If this were a real XML project, you would want to be sure to remove all the spaces between the text within your tafs and the tags themselves. If we querying this XML markup for data, " String " would get returned as the unit of data here rather than "String." The same applies to all your tags below. I didn't stress that here since we aren't creating an actual XML document to query, so it's not an issue necessarily for this post, but you should know in future that you would need to edit all these tags.